exampleCourse/externalGradingFilesCourse/python_autograder/run.sh

```diff
@@ -19,16 +23,31 @@ mkdir $BIN_DIR
 mkdir $OUT_DIR
 
 cp $STUDENT_DIR* $BIN_DIR
-cp $SHARED_DIR* $MERGE_DIR
+cp $AG_DIR* $MERGE_DIR
 cp $TEST_DIR* $MERGE_DIR
 
-# move our run script into its proper location
-cp $SHARED_DIR'run.sh' $JOB_DIR'run.sh'
-
 # we need this to include code as python modules
 # There is already one in the /run directory, but we need one in the /run/bin directory as well
 echo "" > $BIN_DIR/__init__.py
```

Comment:
This definitely did not work without an __init__.py script in the past. Everywhere I searched online in the past mentioned it as well. Does it work for you without one?